tmeister/static/src/toggles/table/toggle-prod-modal.component.js

```diff
@@ -70,25 +68,28 @@ export default function ToggleProdModal (props) {
             </div>
           )
         }
-      </div>
-      <div className={classes.buttonBar}>
-        <SimpleButton
-          actionType="primary"
-          onClick={() => {
-            if (rolling) {
-              performChange(`ROLL:${days}`)
-            } else {
-              performChange()
```

Comment:
I know this is not changed with this particular MR, but it's unclear what `performChange` does differently when passing it a string vs nothing at all.